Pod/Classes/SKYPubsubContainer.h

```diff
@@ -26,6 +26,10 @@ NS_ASSUME_NONNULL_BEGIN
 
 @property (nonatomic, readonly) NSURL *_Nullable endPointAddress;
 
+@property (nonatomic, copy, nullable) void (^onOpenCallback)(void);
```

Comment:
I think the open, close, and error callback should expose as delegate rather than separate callback functions. Are there other projects that do the same thing with separate callbacks?

Reply:
Oh. You are right. Setting a callback is not iOS style. Shall we implement it using `NSNotificationCenter` ?

Reply:
no, use delegate